fearless/Modules/SwapTransactionDetail/SwapTransactionDetailViewLayout.swift

```diff
@@ -176,10 +229,44 @@ final class SwapTransactionDetailViewLayout: UIView {
             makeCommonConstraints(for: view)
         }
 
-        closeButton.snp.makeConstraints { make in
-            make.leading.trailing.equalToSuperview().inset(UIConstants.bigOffset)
+        transactionHashView.snp.makeConstraints { make in
+            make.height.equalTo(Constants.multiViewHeight)
```

Comment:
I suggest use makeCommonConstraints(for: view) for defining height of each view)